evm/v0.5/testv2/Schnorr/fixtures/dssTest.ts

```diff
@@ -1,11 +1,9 @@
 // This test was generated by by kyber/sign/ethdss/ethdss_test.go, with
 // printTests set to true.
-const dssTest = [
+export const dssTest = [
```

Comment:
What do you think of just adding this into the file it's consumed in? It seems to only be used in one place.